build.sbt

```diff
@@ -132,8 +132,8 @@ inThisBuild(
     resolvers += Resolver.sonatypeRepo("public"),
     dependencyOverrides += V.guava,
     // faster publishLocal:
-    publishArtifact.in(packageDoc) := sys.env.contains("CI"),
-    publishArtifact.in(packageSrc) := sys.env.contains("CI"),
+    publishArtifact.in(Compile / packageDoc) := sys.env.contains("CI"),
```

Comment:
Funily enough, without this change we would try to publish Tests and it would fail currently.